playbooks/gce/openshift-node/config.yml

```diff
@@ -12,6 +12,11 @@
   hosts: "tag_env-host-type-{{ oo_env }}-openshift-master"
   connection: ssh
   user: root
+  gather_facts: yes
+  tasks:
+    - command: 'curl "http://metadata.google.internal/computeMetadata/v1/instance/network-interfaces/0/access-configs/0/external-ip" -H "Metadata-Flavor: Google"'
+      register: output
+    - set_fact: gce_public_ip="{{ output.stdout }}"
```

Comment:
Is the ansible provided gce_public_ip fact not available here? Seems wrong to have to query the metadata to get it.